TaskModel.json

```diff
@@ -7884,6 +7884,25 @@
 				{ "name": "bistroseats" }
 			]
 		},
+		{
+			"group": 1,
+			"name": "FlixTrain von %s nach %s",
+			"service": 1,
+			"descriptions": [
+				"Bringe die Fahrgäste billig durch Deutschland.",
+				"Halte auch in Haneu",
+				"Fahre preiswert und pünktlich von %s nach %s."
+			],
+			"objects": [
+				{
+					"stations": [ "TS", "RH", "FD", "FMZ", "FFS", "FO", "FH", "FFU", "FKW", "HG", "HHI", "HBS", "HWOB", "BSPD", "BL"],
```

Comment:
Frankfurt Süd/FFS gibt es nicht im Spiel 😅

Reply:
Ich bin noch recht neu, wenn ich FFS hinzufügen wollte: Was müsste ich dafür tun?

Reply:
Ich glaub, Frankfurt Süd ist ein bisschen zu nah am Hbf, dass der Sinn ergibt...

Reply:
Ansonsten braucht es für eine neue Haltestelle einen Eintrag in der Station.json und dann Strecken in Path.json.

Die können manuell erstellt werden, oder auch aus CSV-Exporten aus trassenfinder.de.
Dafür gibt es (mindestens?) zwei Tools: 
- https://tcroute.f2k1.de/ (webbasiert) 
- mein Tool-Set (https://github.com/c1710/TrainCompany-tools). Das läuft aber lokal. Mit Visual Studio Code + Docker ginge es auch, oder online auf gitpod.io/#github.com/<dein-nutzername>/TrainCompany-Data

Reply:
Die Rhein-Main-Bahn von FD nach FMZ scheint auch zu fehlen 🤔

Reply:
Ja, auch Deutschland ist noch nicht voillständig, auch wenn es manchmal den Eindruck erweckt 😅

Reply:
Wo wäre denn ein guter Ansatzpunkt, um die Strecke einzufügen?